js/localization/cs.json

```diff
@@ -0,0 +1,239 @@
+
+        "dxDateBox-simulatedDataPickerTitleTime": "Vyberte čas",
+        "dxDateBox-simulatedDataPickerTitleDate": "Vyberte datum",
+        "dxDateBox-simulatedDataPickerTitleDateTime": "Vyberte datum a čas",
+        "dxDateBox-validation-datetime": "Hodnota musí být datum nebo čas",
+
+        "dxFileUploader-selectFile": "Vyberte soubor",
+        "dxFileUploader-dropFile": "nebo přeneste soubor sem",
+        "dxFileUploader-bytes": "bytů",
+        "dxFileUploader-kb": "kb",
+        "dxFileUploader-Mb": "Mb",
+        "dxFileUploader-Gb": "Gb",
+        "dxFileUploader-upload": "Nahrát",
+        "dxFileUploader-uploaded": "Nahráno",
+        "dxFileUploader-readyToUpload": "Připraveno k nahrání",
+        "dxFileUploader-uploadFailedMessage": "Nahrávání selhalo",
+
+        "dxRangeSlider-ariaFrom": "Od",
+        "dxRangeSlider-ariaTill": "Do",
+
+        "dxSwitch-onText": "ZAP",
+        "dxSwitch-offText": "VYP",
+
+        "dxForm-optionalMark": "volitelný",
+        "dxForm-requiredMessage": "{0} je vyžadováno",
+
+        "dxNumberBox-invalidValueMessage": "Hodnota musí být číslo",
+
+        "dxDataGrid-columnChooserTitle": "Výběr sloupců",
+        "dxDataGrid-columnChooserEmptyText": "Přesuňte sloupec zde pro skytí",
+        "dxDataGrid-groupContinuesMessage": "Pokračovat na další straně",
+        "dxDataGrid-groupContinuedMessage": "Pokračování z předchozí strany",
+        "dxDataGrid-groupHeaderText": "Sloučit sloupce",
+        "dxDataGrid-ungroupHeaderText": "Oddělit",
+        "dxDataGrid-ungroupAllText": "Oddělit vše",
+        "dxDataGrid-editingEditRow": "Upravit",
+        "dxDataGrid-editingSaveRowChanges": "Uložit",
+        "dxDataGrid-editingCancelRowChanges": "Zrušit",
+        "dxDataGrid-editingDeleteRow": "Smazat",
+        "dxDataGrid-editingUndeleteRow": "Obnovit",
+        "dxDataGrid-editingConfirmDeleteMessage": "Opravdu chcete smazat tento záznam?",
+        "dxDataGrid-validationCancelChanges": "Zrušit změny",
+        "dxDataGrid-groupPanelEmptyText": "Přeneste hlavičku sloupce zde pro sloučení",
+        "dxDataGrid-noDataText": "Žádná data",
+        "dxDataGrid-searchPanelPlaceholder": "Hledání...",
+        "dxDataGrid-filterRowShowAllText": "(Vše)",
+        "dxDataGrid-filterRowResetOperationText": "Reset",
+        "dxDataGrid-filterRowOperationEquals": "Rovná se",
+        "dxDataGrid-filterRowOperationNotEquals": "Nerovná se",
+        "dxDataGrid-filterRowOperationLess": "Menší",
+        "dxDataGrid-filterRowOperationLessOrEquals": "Menší nebo rovno",
+        "dxDataGrid-filterRowOperationGreater": "Větší",
+        "dxDataGrid-filterRowOperationGreaterOrEquals": "Větší nebo rovno",
+        "dxDataGrid-filterRowOperationStartsWith": "Žačíná na",
+        "dxDataGrid-filterRowOperationContains": "Obsahuje",
+        "dxDataGrid-filterRowOperationNotContains": "Neobsahuje",
+        "dxDataGrid-filterRowOperationEndsWith": "Termine avec",
```

Comment:
Hi, Laupe!
"Termine avec" - is it the right translation? It looks like French :)

Reply:
"Termine avec" is "Končí na".
and there is a typo in "dxDataGrid-filterRowOperationStartsWith": "Začíná na"